sysutils/socat/Portfile

```diff
@@ -22,22 +23,22 @@ long_description        \
  GNU line editor (readline), a program, or a combination of two of these.
 
 categories      sysutils net
+installs_libs   no
 license         {GPL-2 OpenSSLException}
```

Comment:
since you're now including the `openssl` PortGroup but do not specify a version, it will install openssl v3.x. In that case, because of the more liberal license, I doubt the `OpenSSLException` is still needed here?

Reply:
Good point - changed.